geem/static/geem/js/geem_validation.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Functions used by validator tab.
+ */
+
+
+/**
+ * Get grid options for validator tab grid.
+ * @returns {Object} Grid options
+ */
+function get_grid_options() {
```

Comment:
Does AgGrid demand a predefined list of fields, or can you have it just upload whatever columns are in a tsv file? I guess the former?

Reply:
It demands a predefined list of fields. I have been unable to find anything in the documentation to indicate otherwise.